pkg/vmimage/data/script.sh

```diff
@@ -204,6 +206,51 @@ sed -i -e 's/^ResourceDisk.Format=.*/ResourceDisk.Format=n/' /etc/waagent.conf
 rpm -q kernel --last | sed -n '1 {s/^[^-]*-//; s/ .*$//; p}' >/var/tmp/kernel-version
 rpm -q atomic-openshift-node --qf '%{VERSION}-%{RELEASE}.%{ARCH}' >/var/tmp/openshift-version
 
+
+# Image hardening
+
+# accounts_minimum_age_login_defs
+sed -i -e "s/PASS_MIN_DAYS.*/PASS_MIN_DAYS     1/g" /etc/login.defs
+sed -i -e "s/PASS_MAX_DAYS.*/PASS_MAX_DAYS     60/g" /etc/login.defs
+
+# no_empty_passwords
+sed --follow-symlinks -i 's/\<nullok\>//g' /etc/pam.d/system-auth
+sed --follow-symlinks -i 's/\<nullok\>//g' /etc/pam.d/password-auth
+
+# disable_ctrlaltdel_reboot
+systemctl mask ctrl-alt-del.target
+
+# accounts_logon_fail_delay
+echo 'FAIL_DELAY 4' >> /etc/login.defs
+
+# accounts_tmout
+echo -e "\n# Set TMOUT to 600 per security requirements" >> /etc/profile
+echo "TMOUT=600" >> /etc/profile
+
+# accounts_max_concurrent_login_sessions
+echo '*	hard	maxlogins	10' >> /etc/security/limits.conf
+
+# banner_etc_issue
+#sed -i -e '/^#Banner.*/ a \
```

Comment:
should we delete these commented-out lines?

Reply:
My plan was to put these back into place.  I attempted a few image builds and quickly found that I had locked myself.  I made the image a little too secure.  This was my attempt to roll back and figure out which checks were causing ssh to be denied.